dydx/client.py

```diff
@@ -146,38 +146,45 @@ def _operate(
             'ref': ref,
             'value': wei
         }
+        actualWithdrawOrDepositAddress = (
+            consts.PAYABLE_PROXY_ADDRESS
+            if market == consts.MARKET_WETH
+            else otherAddress
+        )
         operations = [{
             'actionType': actionType,
             'accountId': 0,
             'amount': amountField,
             'primaryMarketId': market,
             'secondaryMarketId': 0,
-            'otherAddress':
-                consts.PAYABLE_PROXY_ADDRESS if market == 0 else otherAddress,
+            'otherAddress': actualWithdrawOrDepositAddress,
             'otherAccountId': 0,
             'data': '0x'
         }]
+        txOptions = dict(
+            value=(
+                wei
+                if (isDeposit and market == consts.MARKET_WETH)
```

Comment:
One thing I had on the typescript client was both an ETH and a WETH market, and would only use the proxy for the ETH one, which could be valuable for market makers that already have WETH

Reply:
Sure, can add that in a later change